### PR TITLE
98% fully working IPC, done using using a relay server

### PR DESCRIPTION
**Pull request by CaelumD25**
If you guys could checkout this branch and see if you're able to run it without any issues that would be great!

To install the requirements for the python packages, just write `pip install requirements.txt` when in the root folder.

### CODE REVIEW

**Comment by bmachado18** [comment]
Running all the test cases in one process gives weird behavior, and ends up failing last two test cases. To solve, test each case in isolation. Otherwise the new IPC is working great for me.

just a side note, to get this working:
1. launch the game in godot
2. install dependencies (python -m pip install --upgrade pip and then pip install -r requirements.txt)
3. run the unit tests and observe (and I just commented out each test to isolate, but there is probably a better way)

make sure you use an admin powershell for windows.

**Comment by CaelumD25** [comment]
> 3\. run the unit tests and observe

Opening the tests in an IDE should allow you to run them by pressing the play arrows in the margin on the left

File: relay.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+from flask import Flask, request, jsonify
+import argparse
+
```
Comment (Decruz24) [inline relay.py:4]: Had to import JSON here so connect worked properly for me

File: python/graph.py
Comment (Decruz24) [inline python/graph.py:1]: Ill get started on this next. Going to make a new branch for this

**Review by Decruz24 — CHANGES_REQUESTED** [review]
Looks really good to me! Just needed to import JSON in relay.py

Thanks for doing so much! I have no idea how you figured this out but it works pretty well.

**Pull request merged**